package.json

```diff
@@ -143,6 +143,7 @@
   },
   "homepage": "https://buttercup.pw",
   "dependencies": {
+    "eva-icons": "^1.1.1",
```

Comment:
@sallar This alright?

Reply:
@perry-mitchell I will move it to dev deps myself